Gemfile

```diff
@@ -24,6 +24,8 @@ gem "simple_form"
 gem "title"
 gem "uglifier"
 gem "unicorn"
+gem "oj"
```

Comment:
Have a look at the `oj` readme. I think there's an issue with Rails 4.1 (maybe 4.2?) that requires an additional helper gem `oj_mimic_json' to ensure that its parser is used.

https://github.com/ohler55/oj#rails